src/pipeline.rs

```diff
@@ -236,7 +236,7 @@ fn build<D>(
                     binding: 1,
                     visibility: wgpu::ShaderStage::FRAGMENT,
                     ty: wgpu::BindingType::Sampler {
-                        filtering: false,
+                        filtering: true,
```

Comment:
wgpu-0.8.0 enabled sampler validation, which caught that we were binding a filtering sampler here, where the bind group layout had specified that there was no filtering.